README.md

```diff
@@ -75,12 +63,11 @@ Then, visit http://127.0.0.1:8000/. This page displays a "Congratulations!" mess
 
 
 - **Querying Data**  
-    
-  - Querying API powered through the amazing MongoDB Aggregation Pipeline  
+
   - Supports most functions of the Django QuerySet API  
   - Support Query Annotations and common SQL AGGREGATE operators  
-  - We support foreign keys and execute JOIN operations all in 1 database call  
-  - Through our custom raw\_aggregate call, MQL operations like Vector Search, Atlas Search, and GeoSpatial querying still yield Django QuerySet results\!
+   Support foreign key usage and execute JOIN operations
```

Comment:
I think "Supports" is the verb tense to use. Dash was inadvertently deleted. Add period.